### PR TITLE
docs: add example to accelerate transaction using RBF in cold wallets

### PR DESCRIPTION
**Pull request by rushilbg**
docs: add example to accelerate transaction using RBF in cold wallets

BTC-898

### CODE REVIEW

**Review by mullapudipruthvik — APPROVED** [review]

File: examples/ts/btc/rbf/cold/original-tx-submit.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/**
+ * Send a transaction to be replaced from a multi-sig cold wallet at BitGo.
+ *
+ * Copyright 2024, BitGo, Inc.  All Rights Reserved.
+ */
+import { BitGo, EnvironmentName } from '../../../../../modules/bitgo';
+import { rbfConfig } from './config';
+
+const bitgo = new BitGo({ env: rbfConfig.env as EnvironmentName });
```
Comment (mmcshinsky-bitgo) [inline examples/ts/btc/rbf/cold/original-tx-submit.ts:9]: Use BitGoAPI examples over new BitGo

File: examples/ts/btc/rbf/cold/original-tx.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+/**
+ * Build a transaction to be replaced from a multi-sig cold wallet at BitGo
+ * for offline signing.
+ *
+ * Copyright 2024, BitGo, Inc.  All Rights Reserved.
+ */
+import { BitGo, EnvironmentName } from '../../../../../modules/bitgo';
+import { rbfConfig } from './config';
+
+const bitgo = new BitGo({ env: rbfConfig.env as EnvironmentName });
```
Comment (mmcshinsky-bitgo) [inline examples/ts/btc/rbf/cold/original-tx.ts:10]: Same change here

File: examples/ts/btc/rbf/cold/replacement-tx-submit.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+/**
+ * Send a replacement transaction from a multi-sig cold wallet at BitGo.
+ *
+ * Copyright 2024, BitGo, Inc.  All Rights Reserved.
+ */
+import { BitGo, EnvironmentName } from '../../../../../modules/bitgo';
+import { rbfConfig } from './config';
+
+const bitgo = new BitGo({ env: rbfConfig.env as EnvironmentName });
```
Comment (mmcshinsky-bitgo) [inline examples/ts/btc/rbf/cold/replacement-tx-submit.ts:9]: Same change here

File: examples/ts/btc/rbf/cold/replacement-tx.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
+/**
+ * Build a replacement transaction from a multi-sig cold wallet at BitGo.
+ *
+ * Copyright 2024, BitGo, Inc.  All Rights Reserved.
+ */
+import { BitGo, EnvironmentName, PrebuildTransactionResult } from '../../../../../modules/bitgo';
+import { rbfConfig } from './config';
+
+const bitgo = new BitGo({ env: rbfConfig.env as EnvironmentName });
```
Comment (mmcshinsky-bitgo) [inline examples/ts/btc/rbf/cold/replacement-tx.ts:9]: Same change here

File: modules/sdk-core/src/bitgo/wallet/wallet.ts [hunk 5]
```diff
@@ -2111,7 +2111,9 @@ export class Wallet implements IWallet {
     } else if (!params.txRequestId && ((hasTxHex && hasHalfSigned) || (!hasTxHex && !hasHalfSigned))) {
       throw new Error('must supply either txHex or halfSigned, but not both');
     }
-    return this.sendTransaction(params);
+    // FIXME: Prefer to use the below assertion for more robustness.
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-core/src/bitgo/wallet/wallet.ts:2114]: Can we add a ticket to track this?

File: modules/sdk-core/src/bitgo/wallet/wallet.ts [hunk 6]
```diff
@@ -3425,4 +3427,33 @@ export class Wallet implements IWallet {
     }
     return keychains;
   }
+
+  async enhanceColdPrebuildTransaction(tx) {
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-core/src/bitgo/wallet/wallet.ts:3431]: Missing JSdocs as well as type annotation for the parameter and return value.

**Review by mmcshinsky-bitgo — CHANGES_REQUESTED** [review]
nit: focus commits to `feat(sdk-core)`

**Review by zahin-mohammad — COMMENTED** [review]
Looks like there is already a bit of un-addressed feedback. Feel free to re-request a review when it is ready for review again.

**Pull request closed without merging**